hypervisor/arch/x86/configs/dnv-cb2.config

```diff
@@ -1,6 +1,6 @@
 # Generated by Kconfiglib (https://github.com/ulfalizer/Kconfiglib)
 CONFIG_PARTITION_MODE=y
-CONFIG_BOARD="dnv_cb2"
+CONFIG_BOARD="dnv-cb2"
```

Comment:
We also need to change the file name under partition.

Reply:
@dongyaozu 
please check updated patch.